kanidmd/src/lib/credential/mod.rs

```diff
@@ -214,30 +214,33 @@ impl Password {
 }
 
 #[derive(Clone, Debug)]
-pub struct BackupCode {
-    // TODO: any other attributes?
+pub struct BackupCodes {
+    // WIP_TODO: any other attributes?
```

Comment:
For now this is okay :) but still good for it to be in a struct incase we add moe.